index.html

```diff
@@ -4,106 +4,374 @@
+            erat, sed diam voluptua?
+          </p>
+
+          <button data-js="btn-answer" class="button" type="button">
+            <span data-js="showAnswer">show</span>
+            <span data-js="hideAnswer" class="hidden">hide</span>
+            answer
+          </button>
+          <h2 data-js="Answer-heading" class="bold hidden">Answer</h2>
+          <p data-js="answer-text" class="card-set__blocksatz hidden">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et.
+          </p>
+
+          <ul class="card-set__ul-card">
+            <li class="card-set__tag">#html</li>
+            <li class="card-set__tag">#css</li>
+            <li class="card-set__tag">#grid</li>
+            <li class="card-set__tag">#responsive</li>
+          </ul>
+        </div>
+      </div>
+    </main>
+
+    <main data-js="main-create" class="main">
+      <header><h1 class="headline">Create</h1></header>
+      <fieldset class="create">
+        <label class="create__headline" for="create-question"
+          >Add question
+          <small class="create__headline--small"
+            >(<span data-js="create-amount-add-question">300 </span> charackters
+            left)</small
+          ></label
+        ><br />
+        <textarea
+          data-js="text-addquestion"
+          name="create-question"
+          id="create-question-1"
+          cols="40"
+          rows="8"
+          maxlength="300"
+          placeholder="Add your question here..."
+        ></textarea>
+        <br />
+        <label class="create__headline mt-15" for="create-answer"
+          >Add answer
+          <small class="create__headline--small"
+            >(<span data-js="create-amount-add-answer">500 </span> charackters
+            left)</small
+          ></label
+        ><br />
+        <textarea
+          data-js="text-addanswer"
+          name="create-answer"
+          id="create-answer"
+          cols="40"
+          rows="10"
+          maxlength="500"
+          placeholder="Add your answer here..."
+        ></textarea>
+        <br />
+        <label class="create__headline mt-15" for="create-label"
+          >Add tags
+          <small class="create__headline--small"
+            >(devided by comma)</small
+          > </label
+        ><br />
+        <textarea
+          name="create-label"
+          id="create-label"
+          cols="40"
+          rows="3"
+        ></textarea>
+        <br />
+        <input type="submit" class="button mt-15" />
+      </fieldset>
+    </main>
+
+    <main data-js="main-profile" class="main hidden">
+      <header><h1 class="headline">Profile</h1></header>
+      <div class="profile">
+        <img
+          class="profile_picture"
+          src="https://source.unsplash.com/random/200x200"
+          alt=""
+        />
+        <h2 class="bold blue">John Doe</h2>
+        <p class="card-set__blocksatz profile__text">
```

Comment:
If ure using the card set styling also on other pages, maybe make a utility with a more generally name for it?